cts/CIB.py

```diff
@@ -415,7 +415,7 @@ class CIB20(CIB12):
     version = "pacemaker-2.5"
 
 class CIB30(CIB12):
-    version = "pacemaker-3.0"
+    version = "pacemaker-3.7"
```

Comment:
Should this be 3.8 (or even 3.9, given the next commit)?

Reply:
Schema versions and CRM feature sets are completely independent, it's a coincidence we're at similar points

The schema is 3.7 as of this PR; this code here is what schema CTS uses as validate-with

Reply:
Ah right.  Yeah, the coincidence makes this a little confusing but hopefully the version numbers will diverge soon and it won't be confusing anymore.